hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/shell/TestBulkDeleteCommand.java

```diff
@@ -64,4 +78,52 @@ public void testArguments() throws IOException, URISyntaxException {
         Assertions.assertThat(bulkDeleteCommand.childArgs.get(0)).
                 describedAs("Children arguments must match").isEqualTo(arg2);
     }
+
+    @Test
```

Comment:
I am thinking if we can add some tests for wrong arguments. ?

Reply:
Added a small test for wrong argument in command line? Do we need a test when the child is not contained in the base path?

Reply:
> Do we need a test when the child is not contained in the base path?

Yes this is want I wanted. assert if the correct error is being thrown in this case.